server/initializers/constants.ts

```diff
@@ -428,7 +428,8 @@ const VIDEO_CATEGORIES = {
   15: 'Science & Technology',
   16: 'Animals',
   17: 'Kids',
-  18: 'Food'
+  18: 'Food',
+  19: 'Political'
```

Comment:
do you thing `'Politics'` would sound better and also fit better?
many category's end with an s, so I guess this would be better.
like `'travels'` and not `'traveling'`

Reply:
Done